packages/shared/tests/schema/project/post.test.ts

```diff
@@ -76,4 +77,8 @@ describe('project post schema', () => {
       }).success,
     ).toBe(false);
   });
+
+  it('validates a matching RepoUrl', () => {
+    expect(Schema.project.post.safeParse(validProject.repoURl).success).toBe(true);
```

Comment:
You need to pass a full object to this; look at the tests above for examples. e.g., if you want to override the `description` from the `validProject`, this is what you'd do:
``` typescript
{
    ...validProject,
    description: Array(PostValidation.MIN_DESCRIPTION_LENGTH - 1).fill('A'),
  }
```
you'll need to do something similar and then pass in a good repo url and a bad one and make sure both work as expected